lib/ansible/modules/network/meraki/meraki_snmp.py

```diff
@@ -317,7 +317,7 @@ def main():
                          peer_ips=dict(type='str'),
                          access=dict(type='str', choices=['none', 'community', 'users']),
                          community_string=dict(type='str', no_log=True),
-                         users=dict(type='list', default=None, element='str', options=user_arg_spec),
+                         users=dict(type='list', default=None, elements='str', options=user_arg_spec),
```

Comment:
If `elements='str'`, `options` doesn't make sense. I assume it should have been `elements='dict'` right from the beginning? @kbreit can you take a look?